addons/crm/models/crm_team.py

```diff
@@ -138,6 +139,40 @@ def write(self, vals):
                 })
         return result
 
+    def unlink(self):
+        """ When unlinking, concatenate ``crm.lead.scoring.frequency`` linked to
+        the team into "no team" statistics. """
+        frequencies = self.env['crm.lead.scoring.frequency'].search([('team_id', 'in', self.ids)])
+        if frequencies:
+            existing_noteam = self.env['crm.lead.scoring.frequency'].sudo().search([
+                ('team_id', '=', False),
+                ('variable', 'in', frequencies.mapped('variable'))
+            ])
```

Comment:
Strange use of no sudo, then sudo for the same model.
Is there something specific about the access rules?

Reply:
Sales people can read but not update / unlink / create. I could add a sudo in the first search but it is not necessary, considering standard ACLs.